styles.css

```diff
@@ -399,3 +401,140 @@ span {
 footer {
   padding-top: 20px;
 }
+
+#navbar-grid {
+  display: none;
+}
+
+#desktop-contactform {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  #navbar-grid {
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+  }
+
+  #desktop-navbar {
+    grid-column: 6 / span 6;
+    justify-content: center;
+  }
+
+  #desktop-contactform {
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    background: url(assets/contactbg.PNG);
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    padding-bottom: 200px;
+    margin-top: 60px;
+  }
```

Comment:
- Hey, let's also improve our design here by making sure the background image is well aligned in this form section as seen in the Figma design.


*Figma design*
![image](https://github.com/AbdulSalamAbro/portfolio-website/assets/59397159/bdbb4c54-0c2b-410a-b676-2fc66d4d609c)

*Your design*
![image](https://github.com/AbdulSalamAbro/portfolio-website/assets/59397159/18f74041-0332-43af-9d5e-38e7c40e45d9)